pkg/sql/plan/function/builtin/unary/math.go

```diff
@@ -91,16 +91,16 @@ func Log(vs []*vector.Vector, proc *process.Process) (*vector.Vector, error) {
 	vals := vs[0].Col.([]float64)
 	for i := range vals {
 		if vals[i] == float64(1) {
-			return nil, errors.New("", "Logarithm function base cannot be 1")
+			return nil, moerr.New(moerr.INVALID_ARGUMENT, "Logarithm function base cannot be 1")
 		}
 	}
 	v1, err := math1([]*vector.Vector{vs[0]}, proc, momath.Ln)
 	if err != nil {
-		return nil, err
+		return nil, moerr.New(moerr.INVALID_ARGUMENT, "Logarithm function input cannot be non positive")
 	}
 	v2, err := math1([]*vector.Vector{vs[1]}, proc, momath.Ln)
 	if err != nil {
-		return nil, err
+		return nil, moerr.New(moerr.INVALID_ARGUMENT, "Logarithm function input cannot be non positive")
```

Comment:
seesm return nil, err is the right thing.  Why moerr.New again?

Reply:
i did't get the newest codes when i fixed the problem.